src/pages/index.vue

```diff
@@ -113,12 +113,13 @@
         <p>
           All of our programs exist to help people answer questions with data.
           <strong>If you are starting a data project, we can help.</strong>
-          <Button
-            class="mt-2"
-            intent="primary"
-            onclick="window.open('https://airtable.com/app473MWXVJVaD7Es/shrbFfWk6fjzGnNsk', '_blank')">
-            Open a Data Request
-          </Button>
+          <router-link :to="'/data-request/create'">
+            <Button
+              class="mt-4"
+              intent="primary">
+              Open a Data Request
+            </Button>
+          </router-link>
```

Comment:
Hmmm, do we need to render a button here? If all we need is the styling, we can just add `"pdap-button-primary"` to the class list, to avoid confusing the markup for keyboard nav and screenreader users.

```suggestion
          <router-link class="pdap-button-primary mt-4" :to="'/data-request/create'">
              Open a Data Request
          </router-link>
```